osmosis-installer.py

```diff
@@ -5,17 +5,63 @@
 import readline
 import random
 import argparse
+import sys
 from os import remove
-from sys import argv
 
-remove(argv[0])
-
-parser = argparse.ArgumentParser(description="Use default settings")
-parser.add_argument('-m', action='store_true', help='use default settings with no input for mainnet')
-parser.add_argument('-t', action='store_true', help='use default settings with no input for mainnet')
 
+#remove(argv[0])
+
+class CustomHelpFormatter(argparse.HelpFormatter):
+    def _format_action_invocation(self, action):
+        if not action.option_strings or action.nargs == 0:
+            return super()._format_action_invocation(action)
+        return ', '.join(action.option_strings)
+    def _split_lines(self, text, width):
+        if text.startswith('R|'):
+            return text[2:].splitlines()
+        # this is the RawTextHelpFormatter._split_lines
+        return argparse.HelpFormatter._split_lines(self, text, width)
+fmt = lambda prog: CustomHelpFormatter(prog,max_help_position=30)
+
+osmo_home = subprocess.run(["echo $HOME/.osmosisd"], capture_output=True, shell=True, text=True).stdout.strip()
+
+parser = argparse.ArgumentParser(description="Osmosis Installer",formatter_class=fmt)
+auto = parser.add_argument_group('Automated')
+auto.add_argument('-m', '--mainnet-default', action='store_true', help='R|Use all default settings with no input for mainnet\n ', dest="m")

```

Comment:
q: what is "R" in `'R|Use all..` ?

Reply:
This is from the CustomHelpFormatter which allows formatting (such as new lines) for the help menu (seen by using the -h flag). Without it the help menu is very hard to read.